src/networking/modules/web-node.js

```diff
@@ -44,6 +44,12 @@ function xdr(superagentConstruct: superagent, endpoint: EndpointDefinition, call
 
   let sc = superagentConstruct;
 
+  if (endpoint.getAbortSignal) {
```

Comment:
Since this is not a `getter` function, you can remove the "get" from this name and just leave `abortSignal`